pkg/flags/flags.go

```diff
@@ -247,6 +259,10 @@ func ParseArgs(args []string) (*EnvFlags, error) {
 		flag.BoolVar(&disableGracefulTeardown, disableGracefulTeardownFlag.Name, false, disableGracefulTeardownFlag.Usage)
 	}
 
+	if flag.Lookup(kubeContextFlag.Name) == nil {
```

Comment:
I think this should be checked against the presence of the `-kubeconfig` argument as well.  Without that, this argument doesn't really hold much of a values. Is it not ?

Reply:
I thought we were using the default location at `~/.kube/config`, but I'll add a check